_includes/blog_modal.html

```diff
@@ -0,0 +1,18 @@
+<img class="blog-image" data-toggle="modal" data-target="#{{ include.target }}" src="{{ site.baseurl }}/assets/images/{{ include.image }}" width="100%">
+
+<div class="container">
+    <div class="modal fade" id="{{ include.target }}" role="dialog">
+      <div class="blog-modal modal-dialog modal-lg">
+        <div class="modal-content">
+          <div class="modal-body">
+            <img data-toggle="modal" data-target="#{{ include.target }}" src="{{ site.baseurl }}/assets/images/{{ include.image }}" width="100%">
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
nit - it looks like this div's indentation is still off. Once you update this then this PR should be good to go.